frontend/package.json

```diff
@@ -42,6 +42,7 @@
     "javascript-time-ago": "2.3.8",
     "js-base64": "^3.7.2",
     "js-yaml": "^4.1.0",
+    "jsonpath-plus": "^6.0.1",
```

Comment:
can we move **package.json** and **package-lock.json** to a different commit explaining why we need this dependency?

Reply:
Dependencies shouldn't go in a different commit. Otherwise a bisect may fail.